packages/core/test/stackViewport_cpu_render_test.js

```diff
@@ -55,7 +55,7 @@ function createViewport(renderingEngine, orientation, width, height) {
 }
 
 // For some reason the cpu rendering is not working properly in the CI
```

Comment:
Should the comment go since I think the xdescribe => describe means it is working?